run.sh

```diff
@@ -1,5 +1,7 @@
 mkdir -p cmake/build
-rm cmake/build/config
 pushd cmake/build
-cmake -DCMAKE_PREFIX_PATH=$MY_INSTALL_DIR ../..
+cmake -DgRPC_INSTALL=ON \
+      -DgRPC_BUILD_TESTS=OFF \
```

Comment:
This gives warning while build. I think we don't need these extra flags in our project.